1_fetch.R

```diff
@@ -0,0 +1,26 @@
+source("1_fetch/src/get_nwis_sites.R")
+
+p1_targets_list <- list(
+  # Load harmonized WQP data product
+  tar_target(
+    p1_wqp_data,
+    readRDS(file = "1_fetch/in/DRB.WQdata.rds")),
+  tar_target(
+    p1_nwis_SpC_sites,
+    {
+      dummy <- '2021-11-12'
```

Comment:
I think it will be useful to have this date set within the main targets script. One way to do that: place `dummy_date <- '2021-11-12'` into the main targets script, and change this line to `dummy <- dummy_date`

Reply:
Oh, just thinking about this suggestion, I think that will force a new download to today's date instead of 11-12

Reply:
Good suggestion, that placement makes more sense! I ran into a situation where I updated the data download step to include max SC, but a site within `p1_nwis_SpC_sites` had been taken offline in the meantime (a site with 2 recorded observations, so perhaps it was a preliminary/test run of some kind). So the dummy variable is used to force a rebuild of the nwis site list if `dummy_date` is changed from `2021-11-12`.